wazo_sysconfd/http_server.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2022 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import multiprocessing
+
+from fastapi import FastAPI
+from gunicorn.app.base import BaseApplication
+
+api = FastAPI(title='wazo-sysconfd', openapi_url='/api/api.yml')
+
+
+def ensure_list(obj):
+    if not isinstance(obj, list):
+        return [obj]
+    else:
+        return obj
+
+
+class SysconfdApplication(BaseApplication):
+    def __init__(self, *args, config: dict = None, **kwargs):
+        self.config = config or {}
+        super().__init__(*args, **kwargs)
+
+    def load_config(self):
+        self.cfg.set(
+            'bind',
+            [
+                '{}:{}'.format(
+                    self.config['rest_api']['listen'], self.config['rest_api']['port']
+                )
+            ],
+        )
```

Comment:
I think this would be more readable if split on multiple lines, without indentation, like:
```
address = ...
port = ...
self.cfg.set('bind', [f'{address}:{port}'])
```

Reply:
I agree. Fixed!